### PR TITLE
feat: update user profile

### PR DESCRIPTION
**Pull request by Don-christo**
**Description of PR**

Endpoint for updating a user's profile details and kyc verification

---

Please check all that apply.

#### Checklist:

- [x] Rebased with master
- [x] PR Description provided
- [x] Did you communicate these changes with the affected teams?
- [ ] Did unit tests/integration tests/E2E tests get properly added/updated?
- [ ] Is this change production-ready including testing, feature toggles, configurations, and deployment considerations?

#### Frontend Checklist:

- [ ] Is this responsive and mobile friendly?
- [ ] Did you follow the [ARIA specs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)

#### Backend Checklist:

- [x] Does the endpoint return the correct success/error format?

---

**Screenshots**  
_add relevant screenshots here_

---

**Test Coverage**  
_add relevant screenshots here_

### CODE REVIEW

**Comment by KristofaJosh** [comment]
Task linked: [CU-86bx0nfcx BE-Update user profile](https://app.clickup.com/t/86bx0nfcx)

File: apps/server/src/controllers/userControllers/userUpdateKyc.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      identification_type,
+      identification_number,
+      identification_doc,
+      proof_of_address_doc,
+    } = validationResult.data
+
+    const files = req.files as {
+      [fieldname: string]: Express.Multer.File[]
+    }
+    console.log(files)
```
Comment (polymathmode) [inline apps/server/src/controllers/userControllers/userUpdateKyc.ts:39]: I guess you forgot to remove the logged 'files'

File: apps/server/src/controllers/userControllers/userUpdateKyc.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      )
+      _identification_doc = identificationDocResult.secure_url
+    }
+
+    if (proofOfAddressDocPath) {
+      const proofOfAddressDocResult = await cloudinary.uploader.upload(
+        files['proof_of_address_doc'][0].buffer.toString('base64')
+      )
+      _proof_of_address_doc = proofOfAddressDocResult.secure_url
+    }
```
Comment (polymathmode) [inline apps/server/src/controllers/userControllers/userUpdateKyc.ts:61]: I'm thinking we could use a logical && operator to handle both conditions in the if blocks??

Comment (Don-christo) [reply apps/server/src/controllers/userControllers/userUpdateKyc.ts:61]: I thought of that but what if the user wants to update only one document and not the two?🤷🏻

**Review by KristofaJosh — CHANGES_REQUESTED** [review]

**Review by KristofaJosh — APPROVED** [review]

**Review by pelzman — APPROVED** [review]

**Pull request merged**